gwswlib/hydx.py

```diff
@@ -0,0 +1,242 @@
+            "fieldname": "HydxMaaiveldschematisering",
+            "unit": "-",
+            "type": ('string', 3),
+            "required": True,
+        },
+        {
+            "csvheader": "WOS_OPP",
+            "fieldname": "HydxOppervlakWaterOpStraat",
+            "unit": "m2",
+            "type": ('float', 8.2),
+            "required": False,
+        },
+        {
+            "csvheader": "KNP_MAT",
+            "fieldname": "HydxMateriaalPut",
+            "unit": "-",
+            "type": ('string', 3),
+            "required": True,
+        },
+        {
+            "csvheader": "KNP_VRM",
+            "fieldname": "HydxVormPut",
+            "unit": "-",
+            "type": ('string', 3),
+            "required": True,
+        },
+        {
+            "csvheader": "KNP_BOK",
+            "fieldname": "HydxNiveauBinnenonderkantPut",
+            "unit": "m",
+            "type": ('float', 8.2),
+            "required": True,
+        },
+        {
+            "csvheader": "KNP_BRE",
+            "fieldname": "HydxBreedte_diameterPutbodem",
+            "unit": "mm",
+            "type": ('float', 8),
+            "required": True,
+        },
+        {
+            "csvheader": "KNP_LEN",
+            "fieldname": "HydxLengtePutbodem",
+            "unit": "mm",
+            "type": ('float', 8),
+            "required": False,
+        },
+        {
+            "csvheader": "KNP_TYP",
+            "fieldname": "HydxTypeKnooppunt",
+            "unit": "-",
+            "type": ('string', 3),
+            "required": True,
+        },
+        {
+            "csvheader": "INI_NIV",
+            "fieldname": "HydxInitieleWaterstand",
+            "unit": "m",
+            "type": ('float', 8.2),
+            "required": False,
+        },
+        {
+            "csvheader": "STA_OBJ",
+            "fieldname": "HydxStatusObject",
+            "unit": "-",
+            "type": ('string', 3),
+            "required": False,
+        },
+        {
+            "csvheader": "AAN_MVD",
+            "fieldname": "HydxAannameMaaiveldhoogte",
+            "unit": "-",
+            "type": ('string', 3),
+            "required": False,
+        },
+        {
+            "csvheader": "ITO_IDE",
+            "fieldname": "HydxIdentificatieDefinitieIT_object",
+            "unit": "-",
+            "type": ('string', 10),
+            "required": False,
+        },
+        {
+            "csvheader": "ALG_TOE",
+            "fieldname": "HydxToelichtingRegel",
+            "unit": "-",
+            "type": ('string', 100),
+            "required": False,
+        },
+    ]
+
+    @classmethod
+    def csvheaders(cls):
+        return [field["csvheader"] for field in cls.FIELDS]
+
+    @classmethod
+    def fields(cls):
+        return cls.FIELDS
+
+    def __init__(self, codes):
+        self.node = {}
+        FIELDS = self.fields()
+        for field in self.fields():
+            
+            # set empty values to None
```

Comment:
Dit is leesbaar genoeg in de code. Is de commentaarregel nodig?

Reply:
Je haalt veel commentaarregels weg. Wat moet ik wel aan commentaar toevoegen?